sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -35,8 +35,8 @@ import org.apache.spark.sql.vectorized.{ColumnarBatch, ColumnVector}
 
 object GpuProjectExec extends Arm {
   def projectAndClose[A <: Expression](cb: ColumnarBatch, boundExprs: Seq[A],
-      gpuOpTime: GpuMetric): ColumnarBatch = {
-    val nvtxRange = new NvtxWithMetrics("ProjectExec", NvtxColor.CYAN, gpuOpTime)
+      totalTime: GpuMetric): ColumnarBatch = {
+    val nvtxRange = new NvtxWithMetrics("ProjectExec", NvtxColor.CYAN, totalTime)
```

Comment:
Why was this change made?  This is a utility method that will be called elsewhere, and this shouldn't be passed a total time metric if one exists.  It should be passed a metric designed to measure time spent on the GPU, as the original parameter name implied.

Reply:
Thanks. I had changed this in the first commit and then inadvertently changed it back in the second commit when merging changes from the other PR. Updated now.